### PR TITLE
Feature/mvp items dow UI

### PR DESCRIPTION
**Pull request by ericyoungquist-ccpo**
resolves the following tickets

[AT-8271](https://ccpo.atlassian.net/browse/AT-8271)

### CODE REVIEW

File: src/steps/05-PerformanceRequirements/DOW/OtherOfferings.vue [hunk 1]
```diff
@@ -206,11 +400,26 @@ export default class OtherOfferings extends Vue {
...
+    this.isDatabase = this.serviceGroupOnLoad.toLowerCase() === "database";
+    this.isStorage = this.serviceGroupOnLoad.toLowerCase() === "storage";
+    this.isTraining = this.serviceGroupOnLoad.toLowerCase() === "training";
+
+    this.isSupport = [
+      "advisory_assistance",
+      "help_desk_services",
+      "documentation_support",
+      "general_cloud_support"
+    ].includes(this.serviceGroupOnLoad.toLowerCase());
```
Comment (tonysumner-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/OtherOfferings.vue:420]: remove debugger

**Comment by tonysumner-ccpo** [comment]
**Compute form** 
- [ ] When choosing `Continuity of Operations Planning (COOP)/Disaster Recovery` the delete button disappears.
- [ ] 
image

**Comment by tonysumner-ccpo** [comment]
**Database summary table**
- [ ] Performance is not displayed
image

**Comment by tonysumner-ccpo** [comment]
**Storage summary table**
- [ ] Table didn’t show.
image

**Comment by tonysumner-ccpo** [comment]
- [ ] Had 2 instances of ` General Iaas, Paas, Saas` and then clicked `I don’t need General Iaas, Paas, Saas` button.  When I returned to the summary page, I decided to re-add  `General Iaas, Paas, Saas`.  Clicked to add it and was directed to the form page with all validation errors lit up.  I was able to fill it and system behaved as expected

File: src/steps/05-PerformanceRequirements/DOW/OtherOfferingSummary.vue [hunk 2]
```diff
@@ -229,135 +240,180 @@ export default class OtherOfferingSummary extends Vue {
...
-        const classificationLevels = ClassificationRequirements.selectedClassificationLevels;
-        let classificationLevel = "";
-        if (classificationLevels.length > 1) {
-          const classificationObj = classificationLevels.find(
-            obj => obj.sys_id === instanceClone.classificationLevel
+  
+
+        if (this.isCompute) {
+          const selectedEnv = instanceEnvTypeOptions.find(
+            obj => obj.value === instanceClone.environmentType
```
Comment (tonysumner-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/OtherOfferingSummary.vue:281]: remove

**Review by tonysumner-ccpo — APPROVED** [review]

**Review by tonysumner-ccpo — APPROVED** [review]

**Review by tonysumner-ccpo — APPROVED** [review]

**Review by tonysumner-ccpo — APPROVED** [review]

**Comment by nkingsley-ccpo** [comment]
Also resolves the following items:

- [x] [AT-8473](https://ccpo.atlassian.net/browse/AT-8473)
- [x] [AT-8270](https://ccpo.atlassian.net/browse/AT-8270)
- [x] [AT-8241](https://ccpo.atlassian.net/browse/AT-8241)
- [x] [AT-8255](https://ccpo.atlassian.net/browse/AT-8255)
- [x] [AT-8265](https://ccpo.atlassian.net/browse/AT-8265)

**Pull request merged**